tailwind.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  purge: {
+    content: [
+      './app/**/*.php',
+      './resources/**/*.php',
+      './resources/**/*.vue',
+      './resources/**/*.js',
+    ],
```

Comment:
It may be useful to further configure PurgeCSS here for WordPress. This can help avoid common pitfalls where styling WordPress elements results in your CSS being purged as it's not found in any of the `content` paths.

Potentially something like the above, though i've not altered it specifically for Sage or Soil etc, and any new WordPress classes they add.

```suggestion
    content: [
      './app/**/*.php',
      './resources/**/*.php',
      './resources/**/*.vue',
      './resources/**/*.js',
    ],
    options: {
      safelist: {
        standard: [...wordPressWhitelistPatterns()],
      },
    },
```

```js
/**
 * List of RegExp patterns for PurgeCSS
 * @returns {RegExp[]}
 */
function wordPressWhitelistPatterns() {
  return [
    /^home(-.*)?$/,
    /^blog(-.*)?$/,
    /^archive(-.*)?$/,
    /^date(-.*)?$/,
    /^error404(-.*)?$/,
    /^admin-bar(-.*)?$/,
    /^search(-.*)?$/,
    /^nav(-.*)?$/,
    /^wp(-.*)?$/,
    /^screen(-.*)?$/,
    /^navigation(-.*)?$/,
    /^(.*)-template(-.*)?$/,
    /^(.*)?-?single(-.*)?$/,
    /^postid-(.*)?$/,
    /^post-(.*)?$/,
    /^attachmentid-(.*)?$/,
    /^attachment(-.*)?$/,
    /^page(-.*)?$/,
    /^(post-type-)?archive(-.*)?$/,
    /^author(-.*)?$/,
    /^category(-.*)?$/,
    /^tag(-.*)?$/,
    /^menu(-.*)?$/,
    /^tags(-.*)?$/,
    /^tax-(.*)?$/,
    /^term-(.*)?$/,
    /^date-(.*)?$/,
    /^(.*)?-?paged(-.*)?$/,
    /^depth(-.*)?$/,
    /^children(-.*)?$/,

    /c-btn(--.*)?$/,

    // Gutenberg styles
    /^has(-.*)?$/,
    /^is(-.*)?$/,
    /^acf(-.*)?$/,
  ];
}
```

Reply:
Tailwind's purge won't purge anything other than unused Tailwind selectors.

Reply:
something like this should definitely get documented down the line though